sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/DDLParserSuite.scala

```diff
@@ -1424,6 +1424,30 @@ class DDLParserSuite extends AnalysisTest {
     assert(exc.getMessage.contains("Columns aliases are not allowed in UPDATE."))
   }
 
+  private def basicMergeIntoCommand(firstValue: String, secondValue: String): String =
```

Comment:
this method is not used

Reply:
Done. (I had considered refactoring some MERGE INTO unit testing to share the logical plans, but it turned out to be more confusing than expected, so I skipped that approach.)